### ISSUE TITLE
Add Dynamic or Animated Widgets  #Flutter

### ISSUE THREAD

**Issue by iamprakash13** [iamprakash13/Flutter-Widgets#6]
**Description:**
add dynamic/animated widget that is custom created or already exists
give a small description on readme.md
add some animated widget with its demo on readme.md
some animated platforms for flutter are flare, nemo. [refer](2dimensions.com)

**Comment by StrangeNoob** [comment]
if you can explain more would certainly work on it ?

**Comment by iamprakash13** [comment]
sure, build some dynamic widget in the sense that it should have change view according to callback/data from db/restapi. or animations interactive to user response on screen to make ui pleasant (custom loader, man walking, shapes shrink n change, or any animation created in [2dimensions](https://www.2dimensions.com))

what else you need to know?

**Comment by StrangeNoob** [comment]
I got it !!! I would loved to work on it !!!

**Comment by ZOROsrk** [comment]
can i add Hero animation ?

**Comment by iamprakash13** [comment]
yes add it. add more attractive animation, widgets to repo. @ZOROsrk

**Comment by iamprakash13** [comment]
add dynamic/ animated widgets in this issue. this issue's PR are given first preference. happy hacking!
